pkg/services/store/entity/sqlstash/sql_storage_server_test.go

```diff
@@ -123,6 +126,75 @@ func TestCreate(t *testing.T) {
 	}
 }
 
+func TestList(t *testing.T) {
+	s := setUpTestServer(t)
+
+	tests := []struct {
+		name string
+		key  string
+
+		errIsExpected bool
+	}{
+		{
+			"request with key lister",
+			"/playlist.grafana.app",
```

Comment:
Could potentially move the failing tests that are do with invalid keys to be outside the server test, and immediately alongside key.go (as key_test.go) - meaning, test the `key.Parse` there. Any thoughts there?

Reply:
it probably wouldn't hurt to have explicit key parsing tests there, but I'd keep the cases here to make sure that list properly deals with the returned errors